content/en/docs/concepts/security/index.md

```diff
@@ -545,11 +548,12 @@ an authorization policy to the workloads to enforce access control.
 For workloads without authorization policies applied, Istio doesn't enforce
 access control allowing all requests.
 
-Authorization policies support both `ALLOW` and `DENY` actions. The deny
-policies take precedence over allow policies. If any allow policies are applied
-to a workload, access to that workload is denied by default, unless explicitly
-allowed by the rule in the policy. When you apply multiple authorization
-policies to the same workload, Istio applies them additively.
+Authorization policies support `ALLOW`, `DENY` and `CUSTOM` actions and the evaluation order is `CUSTOM`, `DENY` and
```

Comment:
I would prefer shorter sentence. "Evaluation order is ..."

https://developers.google.com/tech-writing/one/short-sentences